From 78da99d8950928d70a1aedafe7289bd911cb34b8 Mon Sep 17 00:00:00 2001
From: Robert Marko <robert.marko@sartura.hr>
Date: Tue, 5 May 2026 16:09:09 +0200
Subject: [PATCH 803/808] dt-bindings: net: pse-pd: microchip,pd692x0: add
 PD77010

PD77010 is a Gen7 Microchip PSE controller, designed for 802.3bt support.

Signed-off-by: Robert Marko <robert.marko@sartura.hr>
---
 .../devicetree/bindings/net/pse-pd/microchip,pd692x0.yaml        | 1 +
 1 file changed, 1 insertion(+)

--- a/Documentation/devicetree/bindings/net/pse-pd/microchip,pd692x0.yaml
+++ b/Documentation/devicetree/bindings/net/pse-pd/microchip,pd692x0.yaml
@@ -18,6 +18,7 @@ properties:
       - microchip,pd69200
       - microchip,pd69210
       - microchip,pd69220
+      - microchip,pd77010
 
   reg:
     maxItems: 1
